src/Base/PyExport.h

```diff
@@ -68,34 +68,35 @@ class PyObjectBase;
  *  because at construction time the reference counter is already set to 1. If the Python
  *  interpreter stores this object pointer into a local variable and destroys this variable
  *  then the reference counter gets decremented (to 0) and the object gets destroyed automatically.
- *  In case we didn't make this specification and increment the Python object from outside once again then
- *  the reference counter would be set to 2 and there would be no chance to destroy the object again.
+ *  In case we didn't make this specification and increment the Python object from outside once
+ * again then the reference counter would be set to 2 and there would be no chance to destroy the
+ * object again.

```

Comment:
Um... when splitting line endings of doc blocks clang-format does not respect the indentation... line 71 two spaces, lines 72-73 one. I would have expected either to format every single line to one space, or respect two spaces when handling too long lines...

Reply:
The problem might be that the line starts with `*`. I have manually removed the spaces.